### ISSUE TITLE
JSON format fails

### ISSUE THREAD

**Issue by brandonzylstra** [pocke/rpr#28]
# How to reproduce this:

```
$ ruby --version
ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin15]
$ command gem install rpr
$ echo 'puts "Hello world"' > hello.rb
$ rpr hello.rb --parser rubocop
s(:send, nil, :puts,
  s(:str, "Hello world"))
$ rpr hello.rb --formatter json
"(send nil :puts\n  (str \"Hello world\"))"
```

That's not actually JSON.  This may be related to another bug I'm about to report, where it fails to work with Ripper.

**Comment by brandonzylstra** [comment]
The other bug is #29

**Comment by pocke** [comment]
I've fixed this bug in 4e0045e2d3b846077376e6d36a9418722b80b362, and released v1.6.1.
Thank you!
